cumulusci/cumulusci.yml

```diff
@@ -159,6 +159,9 @@ tasks:
             version: latest_beta
             activateRSS: True
         group: Salesforce Packages
+    list_communities:
+        description: Lists Communities for the current org using the Connect API.
+        class_path: cumulusci.tasks.salesforce.ListCommunities
```

Comment:
Let's add `group: Salesforce Communities` to this and the other community-related tasks so that they are grouped together in `cci task list`